overview/default.nix

```
@@ -221,12 +243,61 @@ let
         '';
       many = projects: concatLines (mapAttrsToList one projects);
     };
+
+    demoGlue.one = exampleText: ''
+      # default.nix
+      {
+        ngipkgs ? import (fetchTarball "https://github.com/ngi-nix/ngipkgs/tarball/main") { },
```

Does it make sense to pin the demo examples to the overview version? This guarentees that users won't run untested demos:

```suggestion
        ngipkgs ? import (fetchTarball "https://github.com/ngi-nix/ngipkgs/tarball/${self.rev}") { },
```

---

Oh yes it does!